assets/queries/cloudFormation/aws/ec2_permissive_network_acl_protocols/query.rego

```diff
@@ -14,7 +14,7 @@ CxPolicy[result] {
 		"resourceName": cf_lib.get_resource_name(resource, name),
 		"searchKey": sprintf("Resources.%s.Properties.Protocol", [name]),
 		"issueType": "IncorrectValue",
-		"keyExpectedValue": sprintf("'Resources.%s.Properties.Protocol' should be hould be either 6 (for TCP), 17 (for UDP), 1 (for ICMP), or 58 (for ICMPv6, which must include an IPv6 CIDR block, ICMP type, and code)", [name]),
+		"keyExpectedValue": sprintf("'Resources.%s.Properties.Protocol' should be either 6 (for TCP), 17 (for UDP), 1 (for ICMP), or 58 (for ICMPv6, which must include an IPv6 CIDR block, ICMP type, and code)", [name]),
```

Comment:
I notice a typo in keyActualValue. Can you fix it, please? Thank you so much, in advance.

"protocol different **then** 6" to "protocol different **than** 6"